share/test_helpers.go

```diff
@@ -65,3 +65,24 @@ func RandShares(t *testing.T, total int) []Share {
 
 	return shares
 }
+
+func RandBenchmarkEDS(b *testing.B, size int) *rsmt2d.ExtendedDataSquare {
```

Comment:
All exported functions should have a docstring. Adding detail to confirm what the function is doing.

Reply:
done

Reply:
you can resolve @distractedm1nd